outlines/data-loading.md

```diff
@@ -0,0 +1,33 @@
+# Data Loading and Management
+
+1. Publications + Subscriptions
+  1. What are they? - compare REST endpoint
+  2. How do they work? - talk about briding collections
```

Comment:
Oh, like the conceptual idea that a publication's purpose is to pump data from a server-side collection into the client side collection with the same name (typically).